packages/embark/src/lib/modules/storage/index.js

```diff
@@ -7,7 +7,9 @@ class Storage {
     if (!this.storageConfig.enabled) return;
 
     this.handleUploadCommand();
-    this.addSetProviders();
+    this.addSetProviders(() => {
+      this.embark.events.setCommandHandler("module:storage:initiated", (cb) => { cb(); });
```

Comment:
Is this a "onReady" equivalent? Like, you are taking avantage of the retroactive requests? If so, it's pretty smart, but also very confusing because it looks like a request that does nothing.
Maybe we can change the name to something like `module:storage:onReady`.

Reply:
Yes, that's exactly what it is. `onReady` would probably have been better.